.github/actions/repo-gardening/src/get-next-valid-milestone.js

```diff
@@ -29,7 +29,7 @@ async function getNextValidMilestone( octokit, owner, repo, plugin = 'jetpack' )
 	for await ( const response of responses ) {
 		// Find a milestone which name is a version number
 		// and it's due dates is earliest in a future
-		const reg = new RegExp( plugin + '/d.d' );
+		const reg = new RegExp( plugin + '\\/\\d\\.\\d' );
 		const nextMilestone = response.data
 			.filter( m => m.title.match( reg ) )
 			.sort( ( m1, m2 ) => parseFloat( m1.title ) - parseFloat( m2.title ) )
```

Comment:
This isn't going to work anymore, milestones are no longer float-like. Now they're like "jetpack/9.6" or "beta/2.4.7".

What you'll probably wind up doing is pulling in a library like [compare-versions](https://www.npmjs.com/package/compare-versions), then strip the `${plugin}/` prefix.

Reply:
addressed in 2eec2fe